cmake/KDAB/modules/KDQtInstallPaths.cmake

```diff
@@ -16,8 +16,16 @@ if(NOT DEFINED QT_VERSION_MAJOR)
     message(FATAL_ERROR "Please set QT_VERSION_MAJOR first (ie. set(QT_VERSION_MAJOR 5))")
 endif()
 
-if(TARGET Qt${QT_VERSION_MAJOR}::qmake)
-    get_target_property(QT_QMAKE_EXECUTABLE Qt${QT_VERSION_MAJOR}::qmake LOCATION)
+# can use qtpaths to find version instead of qmake
+# but cmake target Qt5::qtpaths DNE, so use qtpaths for 6 only
+if (QT_VERSION_MAJOR EQUAL 6)
+    set(findqt qtpaths)
+else()
+    set(findqt qmake)
+endif()
+
+if(TARGET Qt${QT_VERSION_MAJOR}::${findqt})
```

Comment:
can we do `if` / `else if` instead of enforcing qtpaths with Qt6?

```
if (qmake)
  // ...
elseif (qtpaths)
  // ...
else
  message("qt not found")
```

Reply:
yes good plan, checking existence for both and going with what is available is much safer.
out of curiosity, why prefer qmake over qtpaths when both are available?

Reply:
Because that's how it was and we haven't received any issues about it besides the linked one, so I'd still keep it at the top and try `qtpaths` only if qmake didn't work